mibidata/mibi_image.py

```diff
@@ -460,4 +461,6 @@ def export_pngs(self, path, size=None):
         for i, label in enumerate(self.channels):
             im = converter(data[:, :, i])
             png_name = label[1] if isinstance(label, tuple) else label
-            skio.imsave(f'{os.path.join(path, png_name)}.png', im)
+            with warnings.catch_warnings():
```

Comment:
If we wanted to be extra careful, we could check specifically for the low-res warning and ignore only in that case.

Reply:
Yeah, that's probably a good idea.